geomstats/geometry/matrices_space.py

```diff
@@ -30,6 +30,39 @@ def belongs(self, point):
         _, mat_dim_1, mat_dim_2 = point.shape
         return mat_dim_1 == self.m & mat_dim_2 == self.n
 
+    @staticmethod
+    def mult(a, b):
+        """
+        Return the product of matrices a and b.
+
+        Parameters
+        ----------
+        a : array-like, shape=[n_samples, dim1, dim2]
+        b : array-like, shape=[n_samples, dim2, dim3]
+
+        Returns
+        -------
+        c : array-like, shape=[n_samples, dim1, dim3]
+        """
+        return gs.matmul(a, b)
+
+    @staticmethod
+    def bracket(a, b):
```

Comment:
maybe call it `lie_bracket` to match Stefan's naming? Or did you choose this because the function is slightly different (it only works at "identity")?

Reply:
we'll just call it commutator